packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -93,6 +107,13 @@ export interface Ec2EnvironmentProps {
    * @default - CONNECT_SSH
    */
   readonly connectionType?: ConnectionType
+
+  /**
+   * The image ID used for creating an Amazon EC2 environment.
+   *
+   * Valid values are: AMAZON_LINUX_2, and UBUNTU_18_04
```

Comment:
Let's remove this line. They're already defined in the enum and this will just provide additional overhead to update in the future.

Reply:
Thanks - have pushed the change